examples/gaussian_process/plot_gpr_dim_selection.py

```diff
@@ -0,0 +1,101 @@
+"""
+=========================================================
+Illustration of SelectDimensionKernel
+=========================================================
+A simple two-dimensional regression example computed in two different ways:
+1. With a product of two RBF kernels on each feature.
```

Comment:
maybe more explicit:
1. With a product of two RBF kernels, where each is applied only to a single feature.